packages/components/react/src/core/CodeMirrorEditor/index.tsx

```diff
@@ -176,13 +178,15 @@ export function CodeMirrorEditor({
   }, [doc]);
 
   return (
-    <div className="h-full relative">
```

Comment:
I've moved the responsability of setting the height to the caller. While working on an example where we wanted a fix height, this felt nicer than to wrap it in a div.

However, maybe in the majority of use cases, `h-full` is preferred. So maybe we should default `className` to `h-full`? I'm not sure.